FlowCryptUI/Views/NavigationBarItemsView.swift

```diff
@@ -69,7 +69,7 @@ public final class NavigationBarItemsView: UIBarButtonItem {
         fatalError("init(coder:) has not been implemented")
     }
 
-    public override var isEnabled: Bool {
+    override public var isEnabled: Bool {
```

Comment:
There are a lot of changes like this one. Was it necessary to do it as a part of the current PR? If not, please do such changes in a separate PR in the future. It seems these changes don't relate to the current PR title and make it too big.

Reply:
Yeah, agree 100%, I started updating these small things when noticed them during development.
But then there were too many of them, so I stopped but it was already too late :)